lte/gateway/deploy/agwc-helm-charts/README.md

```diff
@@ -17,6 +17,13 @@ The following table list the configurable parameters of the agw chart and their
 | `config.domainName` | Orchestrator domain name. | `` |
 | `persistant.name` | Secret name containing agwc certs. | `agwc-claim` |
 
+## Pre-requisites
```

Comment:
Typo: Its "Prerequisites" without the dash

Reply:
Fixed